db/seeds.rb

```diff
@@ -1,308 +1,74 @@
-  # FIXME: This produces a deprecation warning. Could replace it with: .order(Arel.sql('random()'))
-  klass.where(organization: org).limit(1).order(Arel.sql('random()')).first
+  Helpers::SqlHelper.random_record_for_org(klass, org)
 end
 
 # Initial starting qty for our test organizations
-base_items = File.read(Rails.root.join("db", "base_items.json"))
-items_by_category = JSON.parse(base_items)
-
-# Creates the Base Items
-items_by_category.each do |category, entries|
-  entries.each do |entry|
-    BaseItem.find_or_create_by!(name: entry["name"], category: category, partner_key: entry["key"])
-  end
-end
-
-pdx_org = Organization.find_or_create_by!(short_name: "diaper_bank") do |organization|
-  organization.name = "Pawnee Diaper Bank"
-  organization.street = "P.O. Box 22613"
-  organization.city = "Pawnee"
-  organization.state = "Indiana"
-  organization.zipcode = "12345"
-  organization.email = "info@pawneediaper.org"
-end
-Organization.seed_items(pdx_org)
+items_by_category = ItemsSeeder.seed
 
-sf_org = Organization.find_or_create_by!(short_name: "sf_bank") do |organization|
-  organization.name = "SF Diaper Bank"
-  organization.street = "P.O. Box 12345"
-  organization.city = "San Francisco"
-  organization.state = "CA"
-  organization.zipcode = "90210"
-  organization.email = "info@sfdiaperbank.org"
-end
-Organization.seed_items(sf_org)
+OrganizationsSeeder.seed
+pdx_org = Organization.find_by(short_name: 'diaper_bank')
+sf_org  = Organization.find_by(short_name: 'sf_bank')
 
 # Assign a value to some organization items to verify totals are working
 Organization.all.each do |org|
   org.items.where(value_in_cents: 0).limit(10).update_all(value_in_cents: 100)
 end
 
 # super admin
-user = User.create email: 'superadmin@example.com', password: 'password', password_confirmation: 'password', organization_admin: false, super_admin: true
+UserSeeder.seed({ email: 'superadmin@example.com', organization_admin: false, super_admin: true})
 
 # org admins
-user = User.create email: 'org_admin1@example.com', password: 'password', password_confirmation: 'password', organization: pdx_org, organization_admin: true
-user2 = User.create email: 'org_admin2@example.com', password: 'password', password_confirmation: 'password', organization: sf_org, organization_admin: true
+UserSeeder.seed({ email: 'org_admin1@example.com', organization_admin: true }, pdx_org)
+UserSeeder.seed({ email: 'org_admin2@example.com', organization_admin: true }, sf_org)
 
 # regular users
-User.create email: 'user_1@example.com', password: 'password', password_confirmation: 'password', organization: pdx_org, organization_admin: false
-User.create email: 'user_2@example.com', password: 'password', password_confirmation: 'password', organization: sf_org, organization_admin: false
+UserSeeder.seed({ email: 'user_1@example.com', organization_admin: false }, pdx_org)
+UserSeeder.seed({ email: 'user_2@example.com', organization_admin: false }, sf_org)
```

Comment:
If we no longer need references to these, would it be more consistent to put these object creations in the user seeder file?

Reply:
I will